recipes/cmaes/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.15)
+project(PackageTest CXX)
+
+find_package(cmaes CONFIG REQUIRED)
+
+add_executable(example src/example.cpp)
+target_link_libraries(example libcmaes::cmaes)
+target_compile_features(example PUBLIC cxx_std_11)
```

Comment:
The package needs at least c++11 for the template headers.
Is this fine here or should this be moved to the `validate` function of the base profile to just exclude profiles with `cppstd<11`?

Reply:
Both things actually! The best approach is to have the validate() method in the main recipe check for the min cppstd (Check how it is done in the [cmake_package](https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/cmake_package/all/conanfile.py) template), and also keep this so we know it works in the lowest valid cppstd :)

(Feel free to steal more ideas from that template if needed elsewhere too)

Reply:
I added the validate function